File: crates/kebab-parse-code/tests/fixtures/sample.rs
//! sample fixture

use std::fmt;

const ANSWER: u32 = 42;

/// Doc comment on a free fn.
pub fn parse(input: &str) -> usize {
    input.len()
}

pub struct Foo {
    pub n: u32,
}

impl Foo {
    /// method doc
    pub fn double(&self) -> u32 {
        self.n * 2
    }

    fn name() -> &'static str {
        "foo"
    }
}

pub trait Greet {
    fn hello(&self) -> String;
}

mod inner {
    pub fn helper() -> bool {
        true
    }
}

File: crates/kebab-parse-code/tests/fixtures/sample.py
"""sample fixture."""
import os

ANSWER = 42

@staticmethod
def free(x):
    """free fn."""
    return x + 1

class Foo:
    """doc."""
    def double(self, n):
        return n * 2

    @classmethod
    def name(cls):
        return "foo"

class Outer:
    class Inner:
        def helper(self):
            return True

def with_decorator():
    pass
